docs/source/quicktour.rst

```diff
@@ -43,61 +128,174 @@ tokens:
+
+.. code-block:: python
+
+    output = tokenizer.encode("Hello, y'all!", "How are you 😁 ?")
+    print(output.tokens)
+    # ["[CLS]", "Hello", ",", "y", "'", "all", "!", "[SEP]", "How", "are", "you", "[UNK]", "?", "[SEP]"]
+
+You can then check the type IDs attributed to each token is correct with
 
 .. code-block:: python
 
     print(output.type_ids)
     # [0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1]
 
+If you save your tokenizer with :meth:`~tokenizers.Tokenizer.save`, the post-processor will be saved
+along.
+
+Encoding multiple sentences in a batch
+----------------------------------------------------------------------------------------------------
+
+To get the full speed of the 🤗 Tokenizers library, it's best to process your texts by batches by
+using the :meth:`~tokenizers.Tokenizer.encode_batch` method:
+
+.. code-block:: python
+
+    output = tokenizer.encode_batch(["Hello, y'all!", "How are you 😁 ?"])
+
+The output is then a list of :class:`~tokenizers.Encoding` objects like the ones we saw before. You
+can process together as many texts as you like, as long as it fits in memory.
+
+To process a batch of sentences pairs, pass two lists to the
+:meth:`~tokenizers.Tokenizer.encode_batch` method: the list of sentences A and the list of sentences
+B:
+
+.. code-block:: python
+
+    output = tokenizer.encode_batch(
+        ["Hello, y'all!", "How are you 😁 ?"],
+        ["Hello to you too!", "I'm fine, thank you!"]
+    )
+
+When encoding multiple sentences, you can automatically pad the outputs to the longest sentence
+present by using :meth:`~tokenizers.Tokenizer.enable_padding`, with the :obj:`pad_token` and its ID
+(which we can double-check the id for the padding token with
+:meth:`~tokenizers.Tokenizer.token_to_id` like before):
+
+.. code-block:: python
+
+    tokenizer.enable_padding(pad_id=3, pad_token="[PAD]")
+
+We can set the :obj:`direction` of the padding (defaults to the right) or a given :obj:`length` if
+we want to pad every sample to that specific number (here we leave it unset to pad to the size of
+the longest text).
+
+.. code-block:: python
+
+    output = tokenizer.encode_batch(["Hello, y'all!", "How are you 😁 ?"])
+    print(output[1].tokens)
+    # ["[CLS]", "How", "are", "you", "[UNK]", "?", "[SEP]", "[PAD]"]
+
+In this case, the `attention mask` generated by the tokenizer takes the padding into account:
+
+.. code-block:: python
+
+    print(output[1].attention_mask)
+    [1, 1, 1, 1, 1, 1, 1, 0]
+
+.. _pretrained:
+
+Using a pretrained tokenizer
+----------------------------------------------------------------------------------------------------
+
+You can also use a pretrained tokenizer directly in, as long as you have its vocabulary file. For
+instance, here is how to get the classic pretrained BERT tokenizer:
+
+.. code-block:: python
+
+    from tokenizers import ByteLevelBPETokenizer
+
+    tokenizer = BertWordPieceTokenizer("bert-base-uncased-vocab.txt", lowercase=True)
```

Comment:
`ByteLevelBPETokenizer` is imported but the tokenizer is initialized with `BertWordPieceTokenizer`.